packages/plugins/RSS3/src/SNSAdaptor/modals/DetailsModal/index.tsx

```diff
@@ -5,34 +5,39 @@ import { useSingletonModal } from '@masknet/shared-base-ui'
 import type { FeedCardProps } from '../../components/base.js'
 import { CardType } from '../../components/share.js'
 import { FeedDetailsDialog } from './DetailDialog.js'
+import { ScopedDomainsContainer } from '@masknet/web3-hooks-base'
 
 export interface FeedDetailsModalOpenProps
     extends Omit<PropsWithChildren<InjectedDialogProps>, 'open'>,
         Pick<FeedCardProps, 'feed' | 'actionIndex'> {
     type: CardType
+    map?: Record<string, string>
```

Comment:
```suggestion
    scopedDomainsMap: Record<string, string>
```

It would be hard to guess what `map` is to FeedDetailsModal